app/models/project_transfer.rb

```diff
@@ -0,0 +1,26 @@
+class ProjectTransfer < ActiveRecord::Base
+  include RequestAudited
+  audited
+
+  belongs_to :project
+  belongs_to :from_user, class_name: 'User'
+
+  has_many :project_permissions, through: :project
+  has_many :project_transfer_users
+  has_many :to_users, through: :project_transfer_users
```

Comment:
We may need to add a validation on either to_users or project_transfer_users to ensure that they're not empty.